docs/examples/artifacts/TextTranslation.json

```diff
@@ -0,0 +1,315 @@
+{
+    "meta": {
+        "version": "0.2.0",
+        "generator": {
+            "softwareName": "Burrito Factory",
+            "softwareVersion": "0.1",
+            "userName": "Jane Doe"
+        },
+        "uploader": {
+            "softwareName": "Burrito Truck",
+            "softwareVersion": "0.1",
+            "userId": "dbl::5678",
+            "userName": "Josh Buck"
+        },
+        "comments": [
+            "Experimenting with i18n",
+            "Fixed canon before upload. ~Josh"
+        ]
+    },
+    "idServers": {
+        "dbl": "https://thedigitalbiblelibrary.org",
+        "agmt": "http://registry.autographamt.com",
+        "x-atl": "http://atlantisbibleconsortium.net"
+    },
+    "identification": {
+        "systemId": {
+            "dbl":{
+		"id": "0123456789abcdef",
+		"revision":"23"
+	    },
+            "gbc": {
+		"id": "55df02965117ad3f2201309b"
+	    },
+            "paratext": {
+		"id": "2d5220a02a7aaac6bcc2831ae262e9aaca5e1abd"
+	    }
+        },
+        "name": {
+            "en": "Scripture Burrito Demo Text Bible",
+            "fr": "Crêpe mexicaine biblique surdimensionnée (démonstration)"
+        },
+        "description": {
+            "en": "A Demonstration Scripture Burrito containing Text, like Paratext Might One Day Produce"
+        },
+        "abbreviation": {
+            "fr": "CMBS"
+        }
+    },
+    "confidentiality": {
+        "metadata": "unrestricted",
+        "source": "private",
+        "publications": "restricted"
+    },
+    "type": {
+        "flavorType": {
+	    "name": "scripture",
```

Comment:
Funny indent, looks like an accidental tab character.  The indent is also inconsistent further down; maybe you could configure your editor to 4 spaces (or set it to auto-detect).  We could put an `.editorconfig` file in the repo.